src/qrl/core/BlockMetadata.py

```diff
@@ -75,6 +95,13 @@ def update_last_headerhashes(self, parent_last_N_headerhashes, last_headerhash:
     def create(block_difficulty=bytes([0] * 32),
                cumulative_difficulty=bytes([0] * 32),
                child_headerhashes=None):
+        """
+        Used by ChainManager. Only override params for testing purposes.

```

Comment:
Better to talk about what this function does, instead of talking used by whom.

Reply:
New comment is "Creates a new instance of BlockMetadata. Only override params for testing purposes." - but there really isn't much to say about what a create() function does.